### PR TITLE
Fix missing kafka max metrics in Atlas

### PR DESCRIPTION
**Pull request by bondj**
And a circular dependency in Eclipse with the test classes

### CODE REVIEW

File: suro-kafka-producer/src/test/java/org/apache/kafka/common/metrics/KafkaMetricBuilder.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.apache.kafka.common.metrics;
+
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.utils.Time;
+
+public class KafkaMetricBuilder {
+	
+	public static KafkaMetric newKafkaMetric(Object lock, MetricName metricName, Measurable measurable, MetricConfig config, Time time)
```
Comment (stevenzwu) [inline suro-kafka-producer/src/test/java/org/apache/kafka/common/metrics/KafkaMetricBuilder.java]: what? you didn't save any lines of code :)

File: suro-kafka-producer/src/test/java/com/netflix/suro/sink/kafka/ServoReporterTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	        
+	        if(monitor.getConfig().getName().equals("negativeInfinityTest"))
+	        {
+	        	assertEquals(0.0, ((AtomicDouble) monitor.getValue()).get(), 0.0001);
+	        	negativityInfitinty=true;
+	        }
+        }
+		
+		assertTrue("Finite Number", negativityInfitinty);
+        assertTrue("Initialised stat", initialised);
```
Comment (stevenzwu) [inline suro-kafka-producer/src/test/java/com/netflix/suro/sink/kafka/ServoReporterTest.java]: alignment

**Comment by stevenzwu** [comment]
look good to me

File: suro-kafka-producer/src/test/java/org/apache/kafka/common/metrics/KafkaMetricBuilder.java (hunk 1 above)
Comment (bondj) [inline suro-kafka-producer/src/test/java/org/apache/kafka/common/metrics/KafkaMetricBuilder.java]: This is a hack to get around the fact the KafkaMetric only having a package leveled constructor.

**Pull request merged**